runner/operator/roslin_qc_operator/roslin_qc_operator.py

```diff
@@ -0,0 +1,81 @@
+from runner.operator.operator import Operator
+from runner.models import Run, Pipeline, Run,  RunStatus
+from runner.serializers import APIRunCreateSerializer
+from .bin import input
+import datetime
+
+class InvalidPipelineError(Exception):
+    pass
+
+class RoslinQcOperator(Operator):
+    """
+    Operator for Roslin QC pipeline
+    """
+
+    def __init__(self, model, request_id = None, run_ids = None):
+        self.run_ids = run_ids
+        self.runs = None
+        if self.run_ids != None:
+            self.runs = Run.objects.filter(id__in = self.run_ids)
+        # TODO: add support for things like FileMetadata's
+        # TODO: add support for multiple request ids
+
+        Operator.__init__(self, model, request_id = request_id, run_ids = run_ids)
+
+    def get_pipeline_id(self):
+        return "9b7f2ac8-03a5-4c44-ae87-1d9f6500d19a"
```

Comment:
I don't think we need this anymore as well.

Reply:
I tried to remove it and things started breaking. Not sure why I think it has to do with the recent changes to how Operators work.